### ISSUE TITLE
[BUG] Home page not available

### ISSUE THREAD

**Issue by tyz126** [Mahmud0808/Iconify#678]
### Describe the bug

The home page button is missing.

### Steps to reproduce

1. Just open the app
2. See error

### Expected behavior

The home page button should appear.

### Screenshots

![Screenshot (Nov 14, 2024 5_24_31 PM)](https://github.com/user-attachments/assets/eb5f61b8-fc9b-4bfd-a99a-2c9c05de3a69)

### Device information

- Device: Samsung Galazy Note 8.0 (GT-N5100)
- OS: Lineage OS Android 12
- App version: v6.9.0
- Magisk/KernelSU: magisk canary 28001

**Comment by Mahmud0808** [comment]
This happens when iconify module is installed but there is no overlays found inside. Clear data and reinstall iconify or switch to official magisk/ksu and try again.

**Comment by tyz126** [comment]
That still didn't work

**Comment by Mahmud0808** [comment]
`/data/adb/modules/Iconify/system/product/overlay` check if this directory is empty or not? And try disabling other modules first before installing iconify.

**Comment by tyz126** [comment]
Yes it is empty. And disabling other modules before installing also didn't work.

**Comment by Mahmud0808** [comment]
Are you using official magisk?

**Comment by tyz126** [comment]
Yes

**Comment by Mahmud0808** [comment]
Then I can't think of any other reason why it wouldn't work.

**Comment by mthw0** [comment]
Same issue here, I just installed version 7.0.0 and the main page is missing. I was successfully using the 6.9.0 version.

**Comment by mthw0** [comment]
I went back to 6.9.0 and it's working there.

**Comment by ness-tr** [comment]
My /data/adb/modules/Iconify/system/product/overlay isn't empty, yet I still don't have the homepage and only Xposed section

**Comment by Mahmud0808** [comment]
@tyz126 @mthw0 @ness-tr Kindly follow this https://t.me/DrDsProjectsChat/1/268337
